charts/substrate-faucet/values.yaml

```diff
@@ -125,3 +125,10 @@ ingress:
   ## @param ingress.enabled Specifies whether as Ingress should be created
   ##
   enabled: false
+
+## Database configuration
+##
+cloudsql:
```

Comment:
What kind of data will be stored in SQL? Do you really need CloudSQL, or will just a PostgreSQL container be enough? 
If you don't require backups or a multi-replica database, running PostgreSQL in Kubernetes is preferred since it is easier to maintain and more cost-efficient.

Reply:
Good point. We can easily survive data loss here. 

It'll be used for two things: 
* store historical "drips", to ensure daily drip limits (current functionality)
* as an AMQP-substitute (using pg-boss library). Faucet will be running multiple instances, all of which will produce and consume jobs.

Is there an a chart with postgres in k8s, that I could use as an example?

Reply:
> Is there an a chart with PostgreSQL in k8s, that I could use as an example?

One of the popular Helm charts for PostgreSQL is the Bitnami chart. Here's an example of how to use it:

1. To the end of the file `charts/substrate-faucet/Chart.yaml`, add the following:

```yaml
dependencies:
  - name: postgresql
    version: "12.8.3" 
    repository: "https://charts.bitnami.com/bitnami"
```

2. In the `values.yaml` file, add the following configuration:

```yaml
postgresql:
  global:
    postgresql:
      auth:
        postgresPassword: "Secret!"
  primary:
    persistence:
      storageClass: "ssd-csi"
      size: 20Gi
    resources:
      limits:
        cpu: 1000m
        memory: 2048Mi
      requests:
        cpu: 500m
        memory: 1024Mi
```

You can find more available parameters [here](https://github.com/bitnami/charts/blob/main/bitnami/postgresql/values.yaml). Note that these parameters should be placed under the `postgresql` root key, which is the name of the dependency.